bugwarrior/db.py

```diff
@@ -197,6 +196,24 @@ def find_taskwarrior_uuid(tw, keys, issue):
             )
         )
 
+    # old bugwarrior did not consider completed task so there could be multiple ones matching
+    for service, key_list in six.iteritems(keys):
+        if any([key in issue for key in key_list]):
+            results = tw.filter_tasks({
+                'and': [("%s.is" % key, issue[key]) for key in key_list],
+                'or': [
+                    ('status', 'pending'),
+                    ('status', 'waiting'),
+                    ('status', 'completed')
```

Comment:
Shouldn't we only be checking `completed` issues here? If it were `pending` or `waiting` it would have already been returned.